addons/better-terrain/editor/TerrainUndo.gd

```diff
@@ -160,3 +160,24 @@ func restore_terrain(ts: TileSet, restore: Array) -> void:
 	for i in restore.size():
 		var r = restore[i]
 		BetterTerrain.set_terrain(ts, i, r.name, r.color, r.type, r.categories)
+
+
+var _cur_action_index := 0
```

Comment:
Move these two variables at the top of the file and name them `_current_...` to avoid unnecessary abbreviation.